docs/ref/forms/modelform-meta.txt

```diff
@@ -0,0 +1,283 @@
+==========================
+ModelForm ``Meta`` options
+==========================
+
+.. module:: django.forms.models
+    :synopsis: ModelForm inner Meta class options
+
+
+.. admonition:: Other ModelForm-related documentation
+
+    This document enumerates and describes the options that can be used to
+    structure a :class:`~django.forms.ModelForm` through its inner ``Meta``
+    class. For additional context, examples and use cases for the
+    :class:`~django.forms.ModelForm` class, please refer to the documentation
+    topic :doc:`Creating forms from models </topics/forms/modelforms>` and the
+    reference entry for :doc:`ModelForm functions</ref/forms/models>`.
+
+.. currentmodule:: django.forms
+
+.. class:: ModelFormOptions
+
+The :class:`~django.forms.ModelForm` helper class can be used to build forms
+that closely reflect specific Django models. The exact structure of the form
+produced can be controlled and customized by defining options in an inner
+``Meta`` class.
+
+.. _ref-modelform-meta-modelformoptions:
+
+``Meta`` and ``ModelFormOptions``
+=================================
+
+The options that can be set in a :class:`~django.forms.ModelForm`'s ``Meta``
+class correspond to the attributes of the
+:class:`~django.forms.ModelFormOptions` class. When a
+:class:`~django.forms.ModelForm` is built, the attributes of its inner ``Meta``
+class are used to build a :class:`~django.forms.ModelFormOptions` object.
+
+::
+
+    # Example using some sort of Book model
+    
+    from django.forms import ModelForm
+    from myLibraryApp.models import Book
+
+    class BookForm(ModelForm)
+        class Meta:
+            model = Book
+            fields = [ 'title', 'author' ]
+            help_texts = {
+                'title' : 'The title of the book',
+                'author': 'The author of the book'
+            }
+            # ... other ModelFormOptions attributes
+
+.. admonition:: Troubleshooting Meta class issues
+
+    Internally, a :class:`~django.forms.ModelFormOptions` object is built using
+    Python's built-in :func:`getattr()` function on the inner ``Meta`` class.
+    This means that while any number of attributes can be included in a ``Meta``
+    class, **only attributes that match those in**
+    :class:`~django.forms.ModelFormOptions` **will be used**. When
+    troubleshooting or encountering unexpected behavior, be sure to check for
+    mistyped or misnamed attributes because these will not, in and of
+    themselves, raise an error (though they might cause another error to be
+    raised if they affect a required attribute).
+
```

Comment:
I don't think we need this admonition; as I've pointed out, `ModelFormOptions` is private API

Reply:
Perhaps this is something I might submit as another (relatively easy?) ticket instead, then-- comparing this to `Model.Options` which [guards against nonexistent attributes by raising a `TypeError` should an invalid attribute be supplied](https://github.com/django/django/blob/5a91ad3d7115c692d497663a155edee5ebc8989c/django/db/models/options.py#L221) , `ModelForm.ModelFormOptions` doesn't seem to do that, instead just ignoring invalid attributes.

Otherwise, what would you say to this: I can rework the admonition to remove mention of `ModelFormOptions` (if we go that route, see my reply above), but keep the hint/warning about invalid `Meta` attributes not seeming to raise an error in and of themselves?

Reply:
I think that if documentation is well-written, there shouldn't be a need for a troubleshooting note advising users to investigate how it works behind the scenes. Unless you're considering repurposing the note for something else?

Reply:
I included this note because I found the behaviour unexpected (considering how many other things will raise errors) and I didn't want somebody to have to go to investigate behind the scenes to realize an easy-to-make but potentially hard-to-spot error.

I will rework or remove this note and consider opening a ticket to add an error if an invalid attribute is set.

Reply:
Agree with @jernwerber on opening a new issue to handle when an invalid attribute is set